kube/src/api/object.rs

```diff
@@ -20,11 +20,11 @@ where
     Modified(K),
     /// Resource was deleted
     Deleted(K),
-    /// Resource bookmark
+    /// Resource bookmark. `Bookmark` is a slimmed down `K` due to [#285](https://github.com/clux/kube-rs/issues/285).
     ///
-    /// From [Watch bookmarks](https://kubernetes.io/docs/reference/using-api/api-concepts/#watch-bookmarks)
-    /// NB: This became Beta first in Kubernetes 1.16
-    /// Slimmed down K for Bookmark WatchEvents due to #285
```

Comment:
Was collapsed and confusing.
![image](https://user-images.githubusercontent.com/639336/103120843-4da48680-462e-11eb-9d19-5981ccbad7eb.png)

Couldn't link to `Bookmark` because it's not public.